common/templates/form-wizard.njk

```diff
@@ -87,12 +87,16 @@
 {% endblock %}
 
 {% block contentSidebar %}
-  {% if person %}
+  {% if person or showAllocationDetails %}
```

Comment:
It would be good to have this logic not be specific to "allocation details". If you think about what we're showing it's no longer really "allocation" specific so this variable name feels little misleading too.

Reply:
What would you suggest instead? The situation where we have a move but not a person is completely specific to allocation, so `showMoveDetails` wouldn't be less misleading...

Reply:
I played around and realised this logic isn't actually working correctly either. This is the new journey now:

![image](https://user-images.githubusercontent.com/3327997/83391142-eb163f80-a3ea-11ea-8d4c-d8a717f52d1c.png)

I don't think we need to tie this to a boolean that we need to specifically set on two particular steps. I got this working correctly using this logic:

```nunjucks
{% if person.id or move.id %}
  ...
{% endif %}
```

Reply:
Means we don't need to do all the changes to the controllers.

Reply:
Damn! good spot.

Reply:
Damn! Good spot.

Reply:
This also solves the issue of the naming...

Reply:
Exactly :)

Reply:
Done 👍